.nvmrc

```diff
@@ -1 +1 @@
-lts/*
+16
```

Comment:
can we please create an issue to remind ourselves that we still want to be able to switch to 18 at some point in the future?

Reply:
Done